languages/csharp/exercises/concept/chars/Chars.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+public static class Identifier
+{
+    public static string Clean(string str)
```

Comment:
Would it make sense to name the parameter `identifier`?

```suggestion
    public static string Clean(string identifier)
```